cluster-image-builder/Dockerfile

```diff
@@ -9,6 +10,14 @@ USER ray
 # Copy requirements file
 COPY requirements.txt .
 
+SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
```

Comment:
Maybe we can maintain multiple `requirements.txt` like [this](https://github.com/vllm-project/vllm/tree/main/requirements) to handle the complexity of different accelerators?